src/NServiceBus.AcceptanceTests/Sagas/When_handling_a_message_with_both_handler_and_timeout_handler.cs

```diff
@@ -0,0 +1,97 @@
+﻿namespace NServiceBus.AcceptanceTests.Sagas
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_handling_a_message_with_both_handler_and_timeout_handler : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_not_invoke_timeout_handler()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<TimeoutSagaEndpoint>(g => g.When(session => session.SendLocal(new StartSagaMessage
+                {
+                    SomeId = Guid.NewGuid()
+                })))
+                .Done(c => c.TestComplete)
```

Comment:
why do we need the CompleteTest message? The framework should make sure that we complete all inflight messages before ending the test and that potentially raised exceptions are thrown. Isn't this the case?